vectornav/src/vectornav.cc

```diff
@@ -483,6 +488,16 @@ class Vectornav : public rclcpp::Node
     }
   }
 
```

Comment:
Please add an appropriate doxygen-style comment block for this function.

Reply:
Done